File: AtlasWeb/Core/Models/NewMaterialSelectComponentViewModel.cs
using System;
using System.Collections.Generic;
using TTDataDictionary;
using TTObjectClasses;

namespace Core.Models
{
    public class NewMaterialSelectComponentViewModel
    {

    }

    public class NewMaterialSelectDTO
    {
        public Material Material { get; set; }
        public StockCard StockCard { get; set; }
        public StockLevelType StockLevelType { get; set; }
        public Currency StoreStock { get; set; }
        public Currency DestinationStoreStock { get; set; }
        public Currency DestinationStoreMaxLevel { get; set; }
        public double? VatRate { get; set; }
    }

    [Serializable]
    public class MaterialSelectorInput
    {
        public MKYS_EMalzemeGrupEnum? MaterialGroup { get; set; }
        public DateTime? TicketDate { get; set; }
        public TransactionTypeEnum? TransactionType { get; set; }
        public Guid? StoreID { get; set; }
    }
}

File: AtlasWeb/Core/wwwroot/app/Logistic/Models/NewMaterialSelectComponentViewModel.ts
import { BaseModel } from 'Fw/Models/BaseModel';
import { Guid } from 'NebulaClient/Mscorlib/Guid';
import { Material, StockCard, StockLevelType } from 'NebulaClient/Model/AtlasClientModel';
import { Type } from 'NebulaClient/ClassTransformer';
export class NewMaterialSelectComponentViewModel extends BaseModel {

}

export class NewMaterialSelectDTO {
    public Material: Material;
    public StockCard: StockCard;
    public StockLevelType: StockLevelType;
    public StoreStock :number;
    public DestinationStoreStock:number;
    public DestinationStoreMaxLevel: number;
    public VatRate:number;
}

